### PR TITLE
Move Library Service to Jellyfin.API

### PR DESCRIPTION
**Pull request by crobibero**
Part of #2872 

This also adds a policy to require the `PermissionKind.EnableContentDownloading` permission

### CODE REVIEW

File: Jellyfin.Api/Controllers/LibraryController.cs [hunk 1]
```diff
@@ -0,0 +1,1021 @@
...
+        /// <summary>
+        /// Get the original file of an item.
+        /// </summary>
+        /// <param name="itemId">The item id.</param>
+        /// <response code="200">File stream returned.</response>
+        /// <response code="404">Item not found.</response>
+        /// <returns>A <see cref="FileStreamResult"/> with the original file.</returns>
+        [HttpGet("/Items/{itemId}/File")]
+        [Authorize(Policy = Policies.DefaultAuthorization)]
+        public ActionResult GetFile([FromRoute] Guid itemId)
```
Comment (Ullmie02) [inline Jellyfin.Api/Controllers/LibraryController.cs:105]: `[ProducesResponseType(StatusCodes.200Ok)]` is missing at every method/endpoint

File: Jellyfin.Api/Controllers/LibraryController.cs [hunk 2]
```diff
@@ -0,0 +1,1021 @@
...
+        /// Gets a list of physical paths from virtual folders.
+        /// </summary>
+        /// <response code="200">Physical paths returned.</response>
+        /// <returns>List of physical paths.</returns>
+        [HttpGet("/Library/PhysicalPaths")]
+        [Authorize(Policy = Policies.RequiresElevation)]
+        public ActionResult<IEnumerable<string>> GetPhysicalPaths()
+        {
+            return Ok(_libraryManager.RootFolder.Children
+                .SelectMany(c => c.PhysicalLocations));
```
Comment (Ullmie02) [inline Jellyfin.Api/Controllers/LibraryController.cs:473]:
```suggestion
            return _libraryManager.RootFolder.Children
                .SelectMany(c => c.PhysicalLocations);
```

Comment (crobibero) [reply Jellyfin.Api/Controllers/LibraryController.cs:473]: This needs to be wrapped in an `Ok` as it actually returns `IEnumerable`

**Comment by barronpm** [comment]
Looks good, once the builds are fixed

**Pull request merged**